Promact.Oauth.Server/src/Promact.Oauth.Server/Exception Handler/SlackUserNotFound.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Promact.Oauth.Server.Exception_Handler
+{
+    public class SlackUserNotFound : Exception
+    {
+        /// <summary>
+        /// Default constructor
+        /// </summary>
+        public SlackUserNotFound() : base()
```

Comment:
Means , you ask about summary comments or to add the new constructor with message parameter and then to send the message in an exception ?